frontend/src/components/PsyListing/PublicPsychologistProfile.jsx

```diff
@@ -101,33 +87,39 @@ const PublicPsychologistProfile = () => {
           </Row>
           <Row className={styles.psyInfo}>
             <Col n="md-6 sm-12">
-              {fields.map(field => (
-                <div key={field.name} className={styles.field} data-test-id="psy-info">
-                  <div className={styles.fieldName}>
-                    {field.name}
-                  </div>
-                  <div>
-                    {field.value ? psychologist[field.value] : field.custom(psychologist)}
+              {fields.map(field => {
+                const value = field.value ? psychologist[field.value] : field.custom(psychologist);
+                return value ? (
+                  <div key={field.name} className={styles.field} data-test-id="psy-info">
+                    <div className={styles.fieldName}>
+                      {field.name}
+                    </div>
+                    <div>
+                      {value}
+                    </div>
                   </div>
-                </div>
-              ))}
+                ) : <></>;
+              })}
             </Col>
             <Col
               n="md-6 sm-12"
               className={styles.mapContainer}
             >
-              {coordinates && (
+              {psychologist.longitude && psychologist.latitude && (
               <MapContainer
-                center={[coordinates.lat, coordinates.lon]}
-                zoom={13}
+                center={[psychologist.latitude, psychologist.longitude]}
+                zoom={psychologist.otherLongitude && psychologist.otherLatitude ? 5 : 13}
```

Comment:
I think most of the time the 2 addresses won't be very far. If there are 2 addresses, I would use zoom 10 instead of 5.

Reply:
Carte trello créée pour zoomer en fonction de la distance entre les 2 points.